src/main/java/org/fundacionjala/coding/danielM/bank_ocr/BankOcr.java

```diff
@@ -0,0 +1,170 @@
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Administrator on 3/10/2017.
+ */
+public class BankOcr {
+
+    /**
+     * This enum type will contain the key values.
+     */
+    private enum KEY {
+        ZERO(0), ONE(1), TWO(2), THREE(3),
+        FOUR(4), FIVE(5), SIX(6),
+        SEVEN(7), EIGHT(8), NINE(9);
+
+        /**
+         * Construct for Enum type.
+         *
+         * @param i of int type.
+         */
+        KEY(final int i) {
+        }
+    }
+
+    private Map<Integer, String> stringMap = new HashMap<>();
+    private static final int MODULUS_FACTOR = 11;
+    private static final int CORRECT_ACCOUNT_LENGTH = 9;
+    private static final int MULTIPLY_FACTOR = 9;
+
+    /**
+     * Constructor private.
+     */
+    public BankOcr() {
+        stringMap.put(KEY.ZERO.ordinal(),
+                          " _ "
+                        + "| |"
+                        + "|_|");
+        stringMap.put(KEY.ONE.ordinal(),
+                          "  |"
+                        + "  |"
+                        + "  |");
+        stringMap.put(KEY.TWO.ordinal(),
+                          " _ "
+                        + " _|"
+                        + "|_ ");
+        stringMap.put(KEY.THREE.ordinal(),
+                          "__ "
+                        + " _|"
+                        + "__|");
+        stringMap.put(KEY.FOUR.ordinal(),
+                          "   "
+                        + "|_|"
+                        + "  |");
+        stringMap.put(KEY.FIVE.ordinal(),
+                          " __"
+                        + "|__"
+                        + " __|");
+        stringMap.put(KEY.SIX.ordinal(),
+                          " __"
+                        + "|__"
+                        + "|__|");
+        stringMap.put(KEY.SEVEN.ordinal(),
+                          "__ "
+                        + "  |"
+                        + "  |");
+        stringMap.put(KEY.EIGHT.ordinal(),
+                          " _ "
+                        + "|_|"
+                        + "|_|");
+        stringMap.put(KEY.NINE.ordinal(),
+                          " _ "
+                        + "|_|"
+                        + " _|");
+    }
+
+    /**
+     * @param value used to obtain its int representation
+     * @return "?" if the value is not in range 0-9 otherwise
+     * return a number between 0 and 9.
+     */
+    private String getKey(final String value) {
+        String key = "?";
+        for (Map.Entry<Integer, String> entry : stringMap.entrySet()) {
+            if (entry.getValue().equalsIgnoreCase(value)) {
+                key = entry.getKey().toString();
+            }
+        }
+        return key;
+    }
+
+    /**
+     * This method determines if an scanned account is legible.
+     *
+     * @param account image of type String.
+     * @return true or false
+     */
+    private boolean isLegible(final String account) {
```

Comment:
I guess you are asking if the account ocr contains a "?" to know if is legible or not.
if that is the casel, this method can be improved.
```java
    return account.contains("?");
```